src/functional/ffront/source_utils.py

```diff
@@ -32,7 +33,7 @@
 
 def make_source_definition_from_function(func: Callable) -> SourceDefinition:
     try:
-        filename = inspect.getabsfile(func) or MISSING_FILENAME
+        filename = os.path.realpath(inspect.getabsfile(func)) or MISSING_FILENAME
```

Comment:
`Pathlib.resolve()` should be equivalent and it is nowadays recommended to use `Pathlib` instead of `os.path` when possible.

Reply:
I changed it to use 'pathlib', but converted it back to 'str'. We may be better off leaving at as Path, what do you think?

Reply:
Yeah, I hadn't thought about it but it might be more reasonable to leave the data as an actual `Path` instance, although you would need then to change the type of the attribute in the `SourceDefinition` class

Reply:
It seems like it's not necessarily a path: https://github.com/GridTools/gt4py/blob/3f261dde9f03590ffdfc7f7b8bf92ce64e4fedc1/tests/eve_tests/unit_tests/test_type_definitions.py#L77. It's probably better to just leave it as is.